src/dataflow.py

```diff
@@ -306,11 +306,11 @@ def stack_analysis(cfg: tac_cfg.TACGraph) -> bool:
                                                    for block in cfg.blocks}
                     if settings.clamp_large_stacks:
                         unmod_stack_changed_count = 0
-                        for succ in curr_block.succs:
+                        for succ in sorted(curr_block.succs):
```

Comment:
Actually, probably don't need this one. I'll go through and remove the overly cautious ones